tools/src/wyvern/tools/tests/effectSeparation/AnnotatedFile.wyt

```diff
@@ -0,0 +1,2 @@
+resource type AnnotatedFile
```

Comment:
Why are the types and the modules implementing them have "file" in their names? They don't implement the file functionality. Consider either renaming the types and the modules or changing them to implement file-related behavior.